content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -336,7 +336,20 @@ workflows:
       condition: not event.pull_request.draft
 {{< /highlight >}}
 
-**Example 2**. This build is triggered on adding a label to a pull request but will continue only if the label added was anything else than "codemagicTest":
+**Example 2**. Use built-in environment variables in the condition. This build will continue only if the source branch is "master." In other words, it will skip the build if the source branch of the pull request is anything other than "master," regardless of the destination branch.
```

Comment:
fyi @dtrdic destination branch is master and source branch is the branch that is created for PR